test/services/rulesService.test.js

```diff
@@ -4,12 +4,12 @@ describe('rulesService', () => {
   test('rules service returns a rules object containing an array', async () => {
```

Comment:
The test description and assertion don't match. The assertions is expecting just an array, not an object containing an array.

Reply:
Fixed